cleanlab_studio/internal/util.py

```diff
@@ -63,3 +64,131 @@ def check_none(x: Any) -> bool:
 
 def check_not_none(x: Any) -> bool:
     return not check_none(x)
+
+
+def _get_autofix_default_params() -> dict:
+    """returns default percentage-wise params of autofix"""
+    return {
+        "ambiguous": 0.0,
+        "label_issue": 0.5,
+        "near_duplicate": 0.2,
+        "outlier": 0.5,
+        "confidence_threshold": 0.95,
+    }
+
+
+def _get_autofix_defaults(cleanset_df: pd.DataFrame) -> dict:
+    """
+    Generate default values for autofix parameters based on the size of the cleaned dataset.
+    """
+    default_params = _get_autofix_default_params()
+    default_values = {}
+
+    for param_name, param_value in default_params.items():
+        if param_name != "confidence_threshold":
+            num_rows = cleanset_df[f"is_{param_name}"].sum()
+            default_values[f"drop_{param_name}"] = math.ceil(num_rows * param_value)
+        else:
+            default_values[f"drop_{param_name}"] = param_value
+    return default_values
+
+
+def _get_top_fraction_ids(
```

Comment:
TODO: Studio team should move this function to backend of the app so it happens on server (eventually should be used in web app too)